state/txindex/kv/kv_test.go

```diff
@@ -51,10 +67,10 @@ func TestTxSearch(t *testing.T) {
 	allowedTags := []string{"account.number", "account.owner", "account.date"}
 	indexer := NewTxIndex(db.NewMemDB(), IndexTags(allowedTags))
 
-	txResult := txResultWithTags([]cmn.KVPair{
-		{Key: []byte("account.number"), Value: []byte("1")},
-		{Key: []byte("account.owner"), Value: []byte("Ivan")},
-		{Key: []byte("not_allowed"), Value: []byte("Vlad")},
+	txResult := txResultWithEvents([]abci.Event{
```

Comment:
I feel like maybe we should preserve the original behaviour here of having multiple attributes for a single event (eg. owner, number), and then having multiple events with distinct attributes (eg. (Ivan, 1) and (Gru, 2).

Reply:
Sure, you believe that will provide a better UX?